pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -533,7 +534,7 @@ func (r *ReconcileUserSignup) DeleteMasterUserRecord(mur *toolchainv1alpha1.Mast
 func (r *ReconcileUserSignup) sendDeactivatedNotification(logger logr.Logger, userSignup *toolchainv1alpha1.UserSignup) error {
 	notification := &toolchainv1alpha1.Notification{
 		ObjectMeta: v1.ObjectMeta{
-			Name:      userSignup.Status.CompliantUsername + "-deactivated",
+			Name:      userSignup.Status.CompliantUsername + "-deactivated" + "-" + strconv.FormatInt(v1.Now().UnixNano(), 10),
```

Comment:
Same as above.

Reply:
This is removed in favour of using GenerateName